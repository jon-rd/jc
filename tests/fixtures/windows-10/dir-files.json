[{"date":"03/25/2021","time":"02:38 PM","dir":false,"size":4253,"filename":"docgen.sh","parent":"C:\\Users\\Yoda\\Desktop\\jc","epoch":1616708280},{"date":"03/25/2021","time":"02:36 PM","dir":false,"size":1088,"filename":"LICENSE.md","parent":"C:\\Users\\Yoda\\Desktop\\jc","epoch":1616708160},{"date":"03/25/2021","time":"02:36 PM","dir":false,"size":27063,"filename":"README.md","parent":"C:\\Users\\Yoda\\Desktop\\jc","epoch":1616708160}]
