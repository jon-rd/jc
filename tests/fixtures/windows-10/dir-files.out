 Volume in drive C has no label.
 Volume Serial Number is 246B-3B5A

 Directory of C:\Users\Yoda\Desktop\jc

03/25/2021  02:38 PM             4,253 docgen.sh

 Directory of C:\Users\Yoda\Desktop\jc

03/25/2021  02:36 PM             1,088 LICENSE.md

 Directory of C:\Users\Yoda\Desktop\jc

03/25/2021  02:36 PM            27,063 README.md
               3 File(s)         32,404 bytes
               0 Dir(s)  33,349,652,480 bytes free
